atuin.yaml

```diff
@@ -22,7 +23,7 @@ pipeline:
       expected-commit: a78aaa78e487b2499ffd7eed86bac15aa3df0960
 
   - runs: |
-      cargo build --locked --release
+      cargo auditable build --locked --release
```

Comment:
What can we do to encourage new rust builds to use this instead of `cargo build`? This is great but keeping it going will depend on folks knowing this is an option.

Reply:
I think there may be room here to standardize our rust package builds in a melange pipeline. I can look into that shortly. Otherwise, a nudge somewhere like Chainguard Academy with code guidelines may also be helpful